docker/alpine/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # compile
-FROM golang:1.20.5-alpine3.18 as builder
+FROM golang:1.21.8-alpine3.18 as builder
```

Comment:
is there an alpine 3.19 image? - i can see that 3.19 offers a slightly newer version 
https://pkgs.alpinelinux.org/packages?name=*libimagequant*&branch=v3.19&repo=&arch=&maintainer=
4.2.2-r0 

compared to 
https://pkgs.alpinelinux.org/packages?name=*libimagequant*&branch=v3.18&repo=&arch=&maintainer=
4.2.0-r0

Reply:
Lets check it out...